.github/workflows/publish_docker_base.yml

```diff
@@ -0,0 +1,31 @@
+name: Publish Base Docker Image to GHCR
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'openshift/wps-api-base/docker/Dockerfile'
+      - '.github/workflows/publish_docker_base.yml'
+jobs:
+  build-and-publish:
+    name: Build and Publish Docker Image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+      - name: Login to GitHub Container Registry
+        # this step requires a GH_TOKEN with read:packages write:packages and delete:packages
+        # tested with a classic token with the above permissions
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{github.actor}}
+          password: ${{secrets.GH_TOKEN}}
+      - name: Build and Publish Base Image
+        # note we can remove the build-arg if we are able to use credentials for artifacts.developer.gov.bc.ca
+        # so we can directly consume that as a base image instead of ubuntu:24.04.
+        run: |
+          cd openshift/wps-api-base/docker
+          docker build --build-arg DOCKER_IMAGE=ubuntu:24.04 . --tag ghcr.io/bcgov/wps/wps-api-base:latest
```

Comment:
Can we create a tag this with the current date in `mm-dd-yyyy` format, then tag it as latest so we have dated versions?